src/customJs/socialShareTool/configuration.js

```diff
@@ -0,0 +1,42 @@
+export const socialSettings = [
+  {
+    link: '[[shareInFacebookLink]]',
+    socialImg:
+      'https://app2.dopplerfiles.com/MSEditor/images/color_big_facebook_en.png',
+    altText: 'Facebook',
+    id: 'facebook',
+  },
+  {
+    link: '[[shareInLinkedinLink]]',
+    socialImg:
+      'https://app2.dopplerfiles.com/MSEditor/images/color_big_linkedin_en.png',
+    altText: 'Linkedin',
+    id: 'linkedin',
+  },
+  {
+    link: '[[shareInTwitterLink]]',
+    socialImg:
+      'https://app2.dopplerfiles.com/MSEditor/images/color_big_twitter_en.png',
+    altText: 'Twitter',
+    id: 'twitter',
+  },
+  {
+    link: '[[shareInPinterestLink]]',
+    socialImg:
+      'https://app2.dopplerfiles.com/MSEditor/images/color_big_pinterest_en.png',
+    altText: 'Pinterest',
+    id: 'pinterest',
+  },
+  {
+    link: '[[shareInWhatsappLink]]',
+    socialImg:
+      'https://app2.dopplerfiles.com/MSEditor/images/color_big_whatsapp_en.png',
+    altText: 'Whatsapp',
```

Comment:
Quizá tiene mas sentido usar un nombre diferente.
```suggestion
    name: 'Whatsapp',
```